### PR TITLE
Add CLI wait 1.0.6

### PR DESCRIPTION
**Pull request by qwuae1**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by yanzhudd** [comment]
please fix the linter issue of CI

File: src/vm-repair/azext_vm_repair/custom.py [hunk 1]
```diff
@@ -644,7 +646,20 @@ def reset_nic(cmd, vm_name, resource_group_name, yes=False):
...
+                                .format(g=resource_group_name, nic=primary_nic_name)
+        
+        current_ip_address = _call_az_command(get_current_ip_command)
+        if current_ip_address == swap_ip_address:
+            logger.info('IP address updated successfully to {ip}\n'.format(ip=swap_ip_address))
+            break
+        else:
+            logger.info('IP address not updated yet. Waiting for a few minutes before retrying...\n')
+            time.sleep(300)
+            # Sleep for 5 minutes
```
Comment (zhoxing-ms) [inline src/vm-repair/azext_vm_repair/custom.py:661]: Are you sure the customer needs to wait for such a long time? I'm not sure if the customer has the patience to wait for 5 minutes

**Comment by yanzhudd** [comment]
please add test for this change and run it in live mode to verify if the change meets your expectations.

**Review by yanzhudd — APPROVED** [review]
Looks good to me.

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**